layers/generate_layers.py

```diff
@@ -133,6 +135,47 @@ def makeMRN():
     run(["tippecanoe", f"mrn/mrn.geojson", "-o", f"mrn.pmtiles"])
 
 
+def makeWards():
+    # Remove files from any previous run
+    try:
+        os.remove("boundary_lines.zip")
+        shutil.rmtree("boundary_lines")
+        os.remove("wards.pmtiles")
+    except:
+        pass
+
+    # Get the geopackage
+    run(
+        [
+            "wget",
+            # From https://osdatahub.os.uk/downloads/open/BoundaryLine
+            "https://api.os.uk/downloads/v1/products/BoundaryLine/downloads?area=GB&format=GeoPackage&redirect",
+            "-O",
+            "boundary_lines.zip",
+        ]
+    )
+    run(["unzip", "boundary_lines.zip", "-d", "boundary_lines"])
+
+    # Convert to GeoJSON, projecting to WGS84. Only grab one layer.
+    run(
+        [
+            "ogr2ogr",
+            "-f",
+            "GeoJSON",
+            "boundary_lines/wards.geojson",
+            "-t_srs",
+            "EPSG:4326",
+            "boundary_lines/Data/bdline_gb.gpkg",
+            "-sql",
+            # Just get a few fields from one layer, and filter for England
+            "SELECT Name, Census_Code, geometry FROM westminster_const WHERE Census_Code LIKE 'E%'",
+        ]
+    )
+
+    # Convert to pmtiles
```

Comment:
I also played with topojson output. Without simplification, the file is about 20MB and has to be loaded all at once -- non-starter. With some reasonable topology-preserving simplifications, I got it down to about 1MB. But we can't readily add this layer to maplibre without solving an NPM publishing issue first anyway, so for now, still pmtiles.